Cargo.toml

```diff
@@ -4,9 +4,9 @@ version = "0.4.2"
 authors = ["Alice Cecile <alice.i.cecile@gmail.com>", "Johnathan Kelley <jkelleyrtp@gmail.com>"]
 edition = "2021"
 include = ["src/**/*", "Cargo.toml"]
-description = "High performance & cross-platform Flexbox implementation"
+description = "A flexible, well-crafted, high performance UI layout library"
```

Comment:
Personally, I'm not a big fan of `well-crafted` as that is kinda subjective IMO.

Reply:
(And I'm not sure if we can confidently call it well-crafted before doing some refactors :P)

Reply:
High performance is subjective too :p I'd like to convey a sense of "high code-quality, good docs, solid tests".

Cutting for now though.

Reply:
```suggestion
description = "A flexible high performance UI layout library"
```